tcp_check/datadog_checks/tcp_check/tcp_check.py

```diff
@@ -53,7 +53,8 @@ def _load_conf(self, instance):
                 addr = socket.gethostbyname(url)
                 socket_type = socket.AF_INET
             except Exception:
-                raise BadConfException("URL: %s is not a correct IPv4, IPv6 or hostname" % addr)
+                msg = "URL: %s is not a correct IPv4, IPv6 or hostname" % url
```

Comment:
nit: can you use the string `format` method instead of `%`?

Reply:
Done.

I prefer `.format()` but the line and the rest of the file was already `%`, also with format it is over 79 characters. Let me know if you want me to split it up into multiple lines.